docs/src/componentDocs/DrawerRailItem/playground/DrawerRailItemConfig.tsx

```diff
@@ -0,0 +1,141 @@
+import { ComponentType } from '../../../__types__';
+import * as Colors from '@brightlayer-ui/colors';
+
+export const drawerRailItemConfig: ComponentType = {
+    componentName: 'Drawer Rail Item',
+    props: [
+        {
+            propName: 'condensed',
+            inputType: 'boolean',
+            inputValue: false,
+            propType: 'boolean',
+            helperText: 'Show condensed nav items without labels (rail variant only)',
+            required: false,
+            defaultValue: false,
+        },
+        {
+            propName: 'disableRailTooltip',
+            inputType: 'boolean',
+            inputValue: false,
+            propType: 'boolean',
+            helperText: 'Sets whether to disable the tooltip on hover (applies when rail item is condensed)',
+            required: false,
+            defaultValue: false,
+        },
+        {
+            propName: 'hidden',
+            inputType: 'boolean',
+            inputValue: false,
+            propType: 'boolean',
+            helperText: 'Sets whether to hide the rail item',
+            required: false,
+            defaultValue: false,
+        },
+        {
+            propName: 'icon',
+            inputType: 'select',
+            inputValue: '<Device />',
+            options: ['<Device />', '<Devices />'],
+            propType: 'JSX.Element',
+            helperText: 'A component to render for the left icon',
+            required: true,
+            defaultValue: '<Device />',
+        },
+        {
+            propName: 'itemID',
+            inputType: 'string',
+            inputValue: 'Title',
+            propType: 'string',
+            helperText: `itemID for the 'active' item`,
+            required: true,
+            disable: true,
+        },
+        {
+            propName: 'statusColor',
+            inputType: 'colorPicker',
+            inputValue: '#ffac00',
+            propType: 'string',
+            helperText: 'Status stripe and icon color',
+            required: false,
+        },
+        {
+            propName: 'title',
+            inputType: 'string',
+            inputValue: 'Title',
+            propType: 'string',
+            helperText: 'The text to show on the first line',
+            required: true,
+            disable: true,
```

Comment:
why is this one also disabled?

Reply:
The reason to disable the title string is if we are changing the title then I need to update the `activeItem` of the drawer onChange of Title input. And this needs to be done in SharedDrawer which is difficult. We can do this but I guess the current playground changes working fine to show railItem props. Any thoughts?

Reply:
Doesn't `activeItem` correspond to only `itemID`, not `title`?

Reply:
I missed this comment, didn't get a notification of this comment. To fix the activeItem issue I have made changes in this PR https://github.com/brightlayer-ui/react-component-library/pull/553. If it looks fine I can fix a rail item similar to this PR.